build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -21,11 +21,22 @@ const jsModuleErrorsExtension = '.ddc.js.errors';
 const jsModuleExtension = '.ddc.js';
 const jsSourceMapExtension = '.ddc.js.map';
 const metadataExtension = '.ddc.js.metadata';
+const fullKernelExtension = '.ddc.full.dill';
 
 /// A builder which can output ddc modules!
 class DevCompilerBuilder implements Builder {
   final bool useIncrementalCompiler;
 
+  /// Make ddc generate full dill for libraries it compiles.
```

Comment:
[nit] Should use a noun phrase.

```suggestion
  /// Whether to generate full dill file outputs for each module.
```

Reply:
thanks, fixed